### PR TITLE
allow to use protov1 messages

### PR DESCRIPTION
**Pull request by ankurs**

### CODE REVIEW

File: initializers.go [hunk 1]
```diff
@@ -18,6 +18,7 @@ import (
 	"github.com/go-coldbrew/log/loggers"
 	"github.com/go-coldbrew/log/loggers/gokit"
 	nrutil "github.com/go-coldbrew/tracing/newrelic"
+	protov1 "github.com/golang/protobuf/proto" //nolint:staticcheck
```
Comment (vestor) [inline initializers.go:21]: Did we add a check in DomainEntities to not allow for lint ignores?
```
lint:
      disallow_comment_ignores: true
```

**Comment by kevinjom** [comment]
LGTM

File: initializers.go (hunk 1 above)
Comment (ankurs) [reply initializers.go:21]: no, we have not set `disallow_comment_ignores: true` anywhere in coldbrew

**Review by kevinjom — APPROVED** [review]

**Review by svetha-cvl — COMMENTED** [review]
lgtm as well 👍

**Pull request merged**